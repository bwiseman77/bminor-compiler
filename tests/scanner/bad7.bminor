l3@t 5p3@k 8r0
