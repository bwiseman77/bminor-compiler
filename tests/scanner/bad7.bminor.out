-------------------------------------------
           token type  text
-------------------------------------------
               IDENT  l3
          SCAN ERROR  @
               IDENT  t
     INTEGER_LITERAL  5
               IDENT  p3
          SCAN ERROR  @
               IDENT  k
     INTEGER_LITERAL  8
               IDENT  r0
